tests/test_unified_scoring_integration.py

```
@@ -0,0 +1,313 @@
+            }""",
+            cost=0.001,
+            model="gemini-pro",
+            provider="google"
+        )
+        
+        mock_provider.generate = AsyncMock(
+            side_effect=[qadi_scores_response, evolution_scores_response]
+        )
+        
+        # Test QADI scoring
+        qadi_orchestrator = SimpleQADIOrchestrator(llm_provider=mock_provider)
```

Ruff (0.12.2)

Local variable `qadi_orchestrator` is assigned to but never used

Remove assignment to unused variable `qadi_orchestrator`

(F841)